### ISSUE TITLE
Running examples/tf/trpo_cartpole_recurrent.py throws attribute error

### ISSUE THREAD

**Issue by jonashen** [rlworkgroup/garage#162]
```Traceback (most recent call last):
  File "/anaconda2/envs/garage/lib/python3.5/site-packages/tensorflow/python/framework/ops.py", line 2327, in get_attr
    c_api.TF_OperationGetAttrValueProto(self._c_op, name, buf)
tensorflow.python.framework.errors_impl.InvalidArgumentError: Operation 'NPO/init_opt/policy/dist_info_sym/scan/TensorArrayStack/TensorArrayGatherV3' has no attr named '_XlaCompile'.
```

**Comment by ryanjulian** [comment]
Related to #199

**Comment by CatherineSue** [comment]
Fixed in #272
